### PR TITLE
Added validation to vote.

### PR DESCRIPTION
**Pull request by tjancz**

### CODE REVIEW

File: src/main/java/pl/confitura/jelatyna/voting/VoteController.java [hunk 1]
```diff
@@ -3,19 +3,39 @@
 import static java.util.stream.Collectors.toList;
 
 import java.time.LocalDateTime;
-import java.util.*;
-import java.util.stream.*;
+import java.util.Collections;
```
Comment (margielm) [inline src/main/java/pl/confitura/jelatyna/voting/VoteController.java]: Please change import settings.

File: src/main/java/pl/confitura/jelatyna/voting/VoteController.java [hunk 2]
```diff
@@ -26,12 +46,20 @@
...
+        WebUtils webUtils, LocalValidatorFactoryBean beanValidator) {
         this.presentationRepository = presentationRepository;
         this.voteRepository = voteRepository;
         this.webUtils = webUtils;
+        this.beanValidator = beanValidator;
+    }
+
+    @InitBinder
+    protected void initBinder(WebDataBinder binder) {
+        binder.addValidators(beanValidator);
```
Comment (margielm) [inline src/main/java/pl/confitura/jelatyna/voting/VoteController.java:43]: there is really no simpler way to do this?

File: src/main/java/pl/confitura/jelatyna/voting/VoteController.java (hunk 1 above)
Comment (tjancz) [reply src/main/java/pl/confitura/jelatyna/voting/VoteController.java]: What is preferred one? 
Currently I'm changing to single imports.

File: src/main/java/pl/confitura/jelatyna/voting/VoteController.java (hunk 2 above)
Comment (tjancz) [reply src/main/java/pl/confitura/jelatyna/voting/VoteController.java:43]: I didn't found any simpler way for enabling @Valid annotation with `@RepositoryRestController`

File: src/main/java/pl/confitura/jelatyna/voting/VoteController.java (hunk 1 above)
Comment (margielm) [reply src/main/java/pl/confitura/jelatyna/voting/VoteController.java]: Yup, please do that and push the changes.

File: src/main/java/pl/confitura/jelatyna/voting/Vote.java [hunk 3]
```diff
@@ -1,37 +1,34 @@
...
-    private Integer order;
-    @OneToOne
-    private Presentation presentation;
-    private Integer rate;
-    @JsonIgnore
-    private LocalDateTime voteDate;
-    @JsonIgnore
-    private String client;
+
+  @Id
```
Comment (margielm) [inline src/main/java/pl/confitura/jelatyna/voting/Vote.java]: I am sorry, but just noticed that you have 2 spaces for indentation. We use 4. Please switch and reformat the code.

**Pull request merged**